apps/core/scenes/resultDetail/ResultDetail.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, Text } from '@kiwicom/universal-components';
+import { withNavigation } from '@kiwicom/margarita-navigation';
+
+type Props = {|
+  +id: ?string,
+|};
+
+class ResultDetail extends React.Component<Props> {
```

Comment:
Shouldn't this screen be called `Booking` or `Booking"Something"` (indicating first step of multistep booking process)? There is just result overview on top of this screen and rest is the passengers info.

Reply:
That's something what I was thinking about and I'm not sure about it. 
I kept this name because it would be too similar with screens which we already have in the app (BookingDetailScreen, BookingsListScreen).

It's not perfect, because as you said it would be fine to know from the name that you are in the first step of bigger process.

Reply:
Maybe we should add some folders here as well,

MMB 			- 		Search
BookingDetail   -		BookingResults

WDYT?

Reply:
It would be still confusing when you have it in your editor or you find for files. I would prefer keep it flat. 

What about rename current bookings:
```
BookingDetailScreen -> MMBDetail
BookingsList -> MMBList
```

And this new screen:
```
ResultDetail -> BookingCreate  (or something like that)
```

Reply:
@jvaclavik I totally understand your concern about mixing naming with MMB.

I think that adding folders can work. It will be nice to have next booking steps (payment, summary...) under some namespace. But it can be definitely part of different PR.

Reply:
Even now it's hard to understand it if you are looking for some file:

<img width="604" alt="screenshot 2019-02-21 at 16 35 00" src="https://user-images.githubusercontent.com/858321/53180831-b1a33180-35f6-11e9-9698-3800ee736b6a.png">

Reply:
It doesn't look that confusing to me (...yet 😄). But I totally get your point.
I'm ok with both options (folders, MMBSomething). I think we can merge this PR as it is to move forward. And create new issue and start discussion there or on slack about better structure/naming for this and next booking multistep screens.